react-client/src/components/Modal.jsx

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import styles from '../styles/Modal.module.css';
+import Typography from "@material-ui/core/Typography";
+import Slider from "@material-ui/core/Slider";
```

Comment:
I'm not sure you're allowed to use pre-written components from a framework like Material UI or Bootstrap. I think we're supposed to write everything from scratch--but you should ask a TM.